modules/administration-guide/partials/con_supported-restic-compatible-backup-servers.adoc

```diff
@@ -0,0 +1,20 @@
+[id="supported-restic-compatible-backup-servers_{context}"]
+= Supported Restic-compatible backup servers
+
+{prod-short} uses {prod-short} Operator and integrated Restic for backing up and restoring {prod-short} instances from backup snapshots on a configured backup server. {prod-short} Operator automates the creation of a Restic backup repository on the backup server. To back up, {prod-short} Operator gathers the data needed for a backup snapshot and uses the integrated Restic to create and manage the snapshot. To restore, {prod-short} Operator uses the integrated Restic to retrieve and decrypt the snapshot, and then {prod-short} Operator applies the retrieved data to perform the recovery.
+
+{prod-short} can use the following backup servers that are compatible with the integrated Restic:
+
+SFTP:: See the documentation for the SFTP server solution you plan to use (link:https://www.openssh.com/[OpenSSH] or a derived commercial product) and link:https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#sftp[Restic Docs on SFTP].
+
+Amazon S3:: See the documentation for link:https://docs.aws.amazon.com/s3/index.html[Amazon S3] (or the chosen S3 API compatible storage) and link:https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#amazon-s3[Restic docs on Amazon S3].
+
+REST:: See the README for link:https://github.com/restic/rest-server[Rest Server] and link:https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#rest-server[Restic docs on Rest Server].
+
+For testing the backing up and restoring of {prod-short} instances, {prod-short} offers the xref:backups-of-che-instances-to-the-internal-backup-server.adoc[internal backup server].
```

Comment:
```suggestion
To test backing up and restoring of {prod-short} instances, use the xref:backups-of-che-instances-to-the-internal-backup-server.adoc[internal backup server].
```
- Avoid giving agency to inanimate objects like software. Che can't really "offer" anything. Instead, rephrase so that human beings use the software.